src/SFA.DAS.Admin.Aan.Web/Models/ManageEvent/EventPageEditFields.cs

```diff
@@ -1,6 +1,6 @@
-﻿namespace SFA.DAS.Admin.Aan.Web.Infrastructure;
+﻿namespace SFA.DAS.Admin.Aan.Web.Models.ManageEvent;
 
-public interface IEventPageEditFields
+public class EventPageEditFields
```

Comment:
Should this be an abstract and name indicate a base class `ManageEventViewModelBase` ?